packages/frontend/src/components/Routing.js

```diff
@@ -366,6 +367,15 @@ class Routing extends Component {
 
                         <NetworkBanner account={account} />
                         <NavigationWrapper />
+                        {
```

Comment:
Put the banner right below `<GlobalAlert />` or else error messages show up weirdly on different sized screens. If you move it right below `<GlobalAlert />`, then the error box will show up where it usually does -> on the top right of the screen


<img width="1246" alt="image" src="https://user-images.githubusercontent.com/25015977/192990601-609f5605-1f34-4c3b-ac6f-4fb52d66ef47.png">